reto.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Recursion do
```

Comment:
Consider renaming the module with something more accurate related to the challenge, maybe as SeriesUtils (for example, because we are defining methods for finding certain things along a series of numbers) or any other you consider.